_targets.R

```diff
@@ -0,0 +1,32 @@
+library(targets)
+source("1_fetch/src/fetch_data.R")
+source("2_process/src/process_data.R")
+source("3_visualize/src/visualize_data.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker", "stringr"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch_data(),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
```

Comment:
A note about our group's `targets` naming conventions -- here's a scenario where we could use either an object target or a file target. You've got a file target here. Our team's convention is that when we have a file target, we include the filename extension in the name of the target (e.g., `eval_data_csv`, `figure_1_png`) so that at a glance, anyone using the pipeline knows it's a file target, and to help us remember when  passing that target to a downstream target that it's a file that will need to be read in within any functions to which it is passed.

Reply:
Also - for the sake of still including one object target in your pipeline (since you'll be asked to use one in the next issue), and for practicing switching between the two, go ahead and convert this back into an object target and add back in the `model_summary_results_csv` target that you removed, which will simply save the processed `eval_data` as a file.

Reply:
All set!